serenata_toolbox/datasets/downloader.py

```diff
@@ -38,7 +38,8 @@ class Downloader:
         '2017-03-15-reimbursements.xz',
         '2017-03-20-purchase-suppliers.xz',
         '2017-04-27-companies-no-geolocation.xz',
```

Comment:
I'm just saying the method is called `fetch_latest_backup` IMHO there should only be listed the most recent datasets. Am I wrong on assuming that? Maybe we should come up with a list of all the datasets available on the S3?